server/app/lib/actions/fusor/subscription/manage_manifest.rb

```diff
@@ -69,7 +69,7 @@ def plan(deployment, customer_portal_credentials)
                 download_file_path = deployment.manifest_file
               end
 
-              ::Fusor.log.debug("existing upstream_consumer: #{download_file_path}")
+              ::Fusor.log.info("existing upstream_consumer: #{download_file_path}")
 
```

Comment:
-1 to these three. Leave as debug. They will only add to questions during a deployment.